.gitignore

```diff
@@ -3,3 +3,4 @@ zcli.config.yml
 zcli.data
 tools/npm/node_modules
 .vscode
+zerops.yml
```

Comment:
we might probably ignore both zerops and import yaml files regardless there position like 
```
*zerops.yml
*import.yml
```

Reply:
that's a good idea :)